src/ray/gcs/redis_context.cc

```diff
@@ -431,6 +431,112 @@ void ValidateRedisDB(RedisContext &context) {
   }
 }
 
+bool isRedisSentinel(RedisContext &context) {
+  auto reply = context.RunArgvSync(std::vector<std::string>{"INFO", "SENTINEL"});
+  if (reply->IsNil() || reply->IsError() || reply->ReadAsString().length() == 0) {
+    RAY_LOG(INFO) << "failed to get redis sentinel info, continue as a regular redis.";
```

Comment:
I'd remove this log and add logs inside ConnectRedisCluster and ConnectRedisSentinel like 

```
RAY_LOG(INFO) << "Connect to Redis cluster/sentinel";
```